romancal/datamodels/schemas/core.schema.yaml

```diff
@@ -328,7 +328,7 @@ properties:
             title: Name of the filter element used
             type: string
             enum: [F062, F087, F106, F129, W146, F158, F184, F213, GRISM, PRISM, DARK]
-        required: [detector, optical_element, name]
```

Comment:
Just curious why `optical_element` is not required. What kind of files are we getting that may not have it?

Reply:
Some reference files (gain is an example).